app/models/forecast.rb

```diff
@@ -1,29 +1,28 @@
 class Forecast < ActiveRecord::Base
 
-	extend Enumerize 
+	extend Enumerize
+
   before_save :set_risk_category
 	belongs_to :report
 
 	validates :peak_load, presence: true
 
-	enumerize :risk, in: [:unlikely, :possible, :likely, :unknown], default: :unknown
-
-  def set_risk_category
-    # Always retrieve the latest configuration
-    config = Config.order("created_at ASC").last
-
-    min = config.possible_min
-    max = config.possible_max
+	enumerize :risk, in: [:unlikely, :possible, :likely, :unknown],
+    default: :unknown
 
-    likely = Range.new(min, max)
+  def possible
+    @range ||= Config.latest.possible_range
+  rescue NoMethodError => e
+    Rails.logger.error "No config found, falling back to null. \n #{e.message}"
+    @range = Config.null
+  end
 
-    result = case
-       when likely.include?(peak_load) then "possible"
-       when peak_load < likely.min then "unlikely"
-       when peak_load > likely.min then "likely"
+  def risk_level
+    case
+    when possible.include?(peak_load) then "possible"
+    when peak_load < possible.min     then "unlikely"
+    when peak_load > possible.max     then "likely"
```

Comment:
This was working previously because a "likely" status would evaluate to false on the first two cases, but evaluate to true on the last case, despite the condition only being greater than the minimum of the range. Definitely better to have this more explicitly correct. Thanks!